### ISSUE TITLE
New pypi release including latest fixes

### ISSUE THREAD

**Issue by Zahlii** [eli5-org/eli5#38]
The current pypi release of 0.13 is dated to 2022-05-11. It still is broken with the latest scipy versions, as https://github.com/eli5-org/eli5/commit/c9b0b24e54bc8746f7f22cc06469781e2af53505 was added but never released to the public. Would it be possible to upload a new version to pypi?

**Comment by lopuhin** [comment]
Hi, yes we're planning a new PyPI release with #44 soon

**Comment by lopuhin** [comment]
this was release on PyPI as 0.14.0, closing
